corehq/messaging/scheduling/scheduling_partitioned/models.py

```diff
@@ -246,10 +249,21 @@ class CaseScheduleInstanceMixin(object):
 
     @property
     @memoized
+    def case(self):
+        try:
+            return CaseAccessors(self.domain).get_case(self.case_id)
+        except (CaseNotFound, ResourceNotFound):
+            return None
+
+    @property
+    @memoized
     def recipient(self):
         if self.recipient_type == 'Self':
-            return CaseAccessors(self.domain).get_case(self.case_id)
+            return self.case
```

Comment:
Looks like this will return None where before it was throwing an exception. is that an ok thing to do and the callers expect a possible None value?